app/scripts/proactive/templates/catalog-publish.html

```diff
@@ -28,6 +28,10 @@
 						<div class="pull-left" id="catalog-publish-objects-title" style="font-weight: bold;padding-left: 15px;">Objects</div>
 					</legend>
 					<div style="height: 270px; overflow: auto;">
+						<div id="new-script-group" class="input-group w-100">
+							<input id="new-script-name" type="text" placeholder="new script name" class="form-control" style="display: table-cell">
+							<button id="add-new-script" class="btn btn-default"><i class="fa fa fa-plus"></i></button>
```

Comment:
Since it's the same action, you can use a className new-script-name for both (input and button)

I suggest to wrap them in form tag as we do in publish wf

Reply:
Done. Thank you for the great suggestion.